swap_meet/vendor.py

```diff
@@ -1,2 +1,94 @@
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    '''
+    Vendor class has an attribute named inventory which starts as an empty list by default and complete 
+    methods such as adding, removing, swapping, and figuring out best categories in your inventory and 
+    your friend's inventory.
+    '''
+    
+    def __init__(self, inventory = None):
+        '''
+        Attribute named inventory which starts as an empty list by default, but can be pass in a list.
+        '''
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+
+    
+    def add(self, add_item):
+        '''
+        Instance method that takes in an item and add it to the inventory list and return the item that was added.
+        '''
+        self.inventory.append(add_item)
+        return add_item
+
+    def remove(self, remove_item):
+        '''
+        Instance method that removes an item from the inventory list and return the item that was removed.
+        '''
+        if remove_item in self.inventory:
+            self.inventory.remove(remove_item)
+            return remove_item
+        else:
+            return False
```

Comment:
A more explicit way to `return False` would be to use a guard clause instead doing it in the else block. If you use a guard clause, then you can also unindent the logic for handling a valid case where there is something to remove. 

```python
if remove_item not in self.inventory:
     return False

self.inventory.remove(remove_item)
     return item
```

Another approach we could take is to try to remove the item directly, and handle the ValueError that occurs if it's not there, and return False to handle it (try/except)